k8s/nginx-ingress/stage-platform-ingress-resource.yaml

```diff
@@ -3,6 +3,32 @@ kind: ConfigMap
 data:
   proxy-body-size:  "200m"
   client-max-body-size: "200m"
+  location-snippets: "  
+    auth_request /auth;
+    error_page 500 = @internal_server_error;
+    error_page 404 = @page_not_found;
+    error_page 403 = @access_denied;
+    error_page 401 = @unauthorized;"
+  server-snippets: "
+    location @internal_server_error { return 500 'Something went wrong'; }
+    location @page_not_found { return 404 'Page not found'; }
+    location @access_denied { return 403 'Access denied'; }
+    location @unauthorized { return 401 'Unauthorized'; }
+
+    location = / { auth_request off; return 202; }
```

Comment:
Return 202 for endpoints that do  not need authorization. This is as we await the auth service to automatically handle these scenarios